.circleci/config.yml

```diff
@@ -5,22 +5,22 @@ version: 2.1
 jobs:
   build:
     docker:
-      - image: golang:1.19.4-buster
+      - image: golang:1.22.2-bullseye
```

Comment:
Curious more than anything on why you've chosen `bullseye` rather than `bookworm` here. Bookworm is a more recent version of Debian than Bullseye.

Reply:
Doesn't matter either way though.

Reply:
There wasn't a particular reason just a mistake on my part they should be the same using bookworm